src/derivatives/common-data-types.md

```diff
@@ -252,7 +252,57 @@ A guide for using macros can be found at
 All REQUIRED metadata fields coming from a derivative file’s source file(s) MUST
 be propagated to the JSON description of the derivative unless the processing
 makes them invalid (for example, if a source 4D image is averaged to create a single
-static volume, a `RepetitionTime` property would no longer be relevant).
+static volume, a `RepetitionTime` property would no longer be relevant). This creates
```

Comment:
is a transition missing? Otherwise I am not sure what "this" refers to.

Reply:
more like a bit I wrote then removed incompletely